src/main/java/reposense/util/FileUtil.java

```diff
@@ -64,7 +66,7 @@ public static void zipFoldersAndFiles(List<Path> pathsToZip, Path sourceAndOutpu
      * {@code sourcePath} into {@code outputPath}.
      */
     public static void zipFoldersAndFiles(List<Path> pathsToZip,
-            Path sourcePath, Path outputPath, String... fileTypes) {
+                                          Path sourcePath, Path outputPath, String... fileTypes) {
```

Comment:
Just a question, does it violate our code standard? I kind of forgot.

Reply:
Oh, good catch. I have fixed the indentation. Not sure why it got changed in the first place.